### PR TITLE
Add Textures image processing example

### PR DESCRIPTION
**Pull request by jhones**

### CODE REVIEW

File: src/Raylib.php [hunk 1]
```diff
@@ -6,13 +6,16 @@
 
 use FFI;
 use FFI\CDATA;
+use Psalm\Type;
```
Comment (nawarian) [inline src/Raylib.php]: Unused import. Please remove.

File: src/Types/Image.php [hunk 2]
```diff
@@ -51,11 +51,29 @@ public function toCData(RaylibFFIProxy $ffi): CData
 
     /**
      * @psalm-suppress MixedPropertyAssignment
+     * @psalm-suppress MixedInferredReturnType
      * @psalm-suppress MixedReturnStatement
      * @psalm-suppress MixedInferredReturnType
      */
     public function updateFromStruct(CData $imageStruct): CData
```
Comment (nawarian) [inline src/Types/Image.php]: I have the impression this method should not exist at all. It is not specific to `Image` struct types and only returns a C Address. If you agree, let's please replace all usages of `Image::updateFromStruct()` with `FFI::addr()` and delete this method.

If you don't agree, please just rename this method to something like `getStructAddress()` without receiving `CData` as a parameter.

**Review by nawarian — CHANGES_REQUESTED** [review]
Thanks a lot for your PR! Everything looks great and works well.

I left some comments here before we merge.

File: src/Raylib.php (hunk 1 above)
Comment (jhones) [reply src/Raylib.php]: Removed

File: src/Types/Image.php (hunk 2 above)
Comment (jhones) [reply src/Types/Image.php]: Removed method ```Image::updateFromStruct()``` and replace all method calls for using ```FFI::addr()```

**Pull request merged**